### DISCUSSION TITLE
Is it possible to disable certain checks that are not relevant to my use case?

### DISCUSSION THREAD

**Discussion by tuckerweibell** [ossf/scorecard#3525] [Q&A]
This tool is awesome! However, there are certain checks I'd like to not surface as they are not relevant to my use case. Is it possible to ignore certain checks? Also, can any checks be modified or customized in the workflow?

**Comment by spencerschrock** [comment]
On the CLI you can use the `--checks` flag to specify which a comma separated list of checks you'd like to run. As a workflow, that's not something that's currently supported. https://github.com/ossf/scorecard-action/issues/729 is probably the closest open issue.
